include/jemalloc/internal/witness.h

```diff
@@ -286,6 +287,21 @@ witness_assert_lockless(witness_tsdn_t *witness_tsdn) {
 	witness_assert_depth(witness_tsdn, 0);
 }
 
+static inline void
+witness_assert_min_depth_to_rank(witness_tsdn_t *witness_tsdn,
```

Comment:
Nit: should probably rename or actually take a depth parameter.

Reply:
Good point. How about `witness_assert_min_rank()`? I'm afraid the relative depth isn't very intuitive to the users; it may even change if we remove / reorder locks.

Reply:
`witness_assert_positive_depth_to_rank` maybe?

Reply:
Makes sense. Will change.